pymatgen/entries/computed_entries.py

```diff
@@ -11,26 +11,187 @@
+    energy correction.
+    """
+    def __init__(self, value, name="Manual adjustment", cls="None", description=""):
+        """
+        Args:
+            value: float, value of the energy adjustment in eV
+            name: str, human-readable name of the energy adjustment.
+                (Default: Manual adjustment)
+            cls: str, the name of the Compatibility class used to generate the
+                energy adjustment. (Default: None)
+            description: str, human-readable explanation of the energy adjustment.
+        """
+        self._value = value
+        self.name = name
+        self.cls = cls
+        self.description = description
+
+        @abc.abstractmethod
+        def value(self):
+            """
+            Return the value of the energy correction in eV
+            """
+            return
+
+    def __repr__(self):
+        output = ["{}:".format(self.__class__.__name__),
+                  "  Name: {}".format(self.name),
+                  "  Value: {:.3f} eV".format(self.value),
+                  "  Description: {}".format(self.description),
+                  "  Generated by: {}".format(self.cls.get("@class", self.cls))]
+        return "\n".join(output)
+
+
+class ConstantEnergyAdjustment(EnergyAdjustment):
+    """
+    A constant energy adjustment applied to a ComputedEntry. Useful in energy referencing
+    schemes such as the Aqueous energy referencing scheme.
+    """
+    def __init__(self, value, name="Constant energy adjustment", cls="None", description="Constant energy adjustment"):
+        """
+        Args:
+            value: float, value of the energy adjustment in eV
+            name: str, human-readable name of the energy adjustment.
+                (Default: Constant energy adjustment)
+            cls: str, the name of the Compatibility class used to generate the
+                energy adjustment. (Default: None)
+            description: str, human-readable explanation of the energy adjustment.
+        """
+        description = description + " ({:.3f} eV)".format(value)
+        super().__init__(value, name, cls, description)
+
+    @property
+    def value(self):
+        """
+        Return the value of the energy correction in eV.
+        """
+        return self._value
+
+    @value.setter
+    def value(self, x):
+        self._value = x
+
+
+class ManualEnergyAdjustment(EnergyAdjustment):
+    """
+    A manual energy adjustment applied to a ComputedEntry.
+    """
+    def __init__(self, value):
+        """
+        Args:
+            value: float, value of the energy adjustment in eV
+        """
+        name = "Manual energy adjustment"
+        description = "Manual energy adjustment ({:.3f} eV)".format(value)
+
+        super().__init__(value, name, cls="None", description=description)
+
+    @property
+    def value(self):
+        """
+        Return the value of the energy correction in eV.
+        """
+        return self._value
+
+    @value.setter
```

Comment:
If you make ManualEnergyAdjustment inherit from Constant and add the setter, that will make the separation of capabilities a bit clearer.

Reply:
I shouldn't need a setter for `ManualAdjustment` either, right? Once the value is set it is what it is (other than scaling with composition)